components/twitter/sources/language-codes.mjs

```diff
@@ -1,4 +1,5 @@
-module.exports = [
+/* eslint-disable */
```

Comment:
Just checking, why are you disabling the lint here?

Reply:
I thought it was creating errors, but it's not now, so I removed that line.